File: jobs/2026-05-27__01-46-48/reminder_sleep_workflow_py__StnUDHi/artifacts/user/myproject/activities.py
import os
from temporalio import activity


@activity.defn(name="notify")
async def notify(message: str) -> str:
    log_path = "/workspace/reminder.log"
    os.makedirs("/workspace", exist_ok=True)
    line = f"Notified: {message}"
    with open(log_path, "a") as f:
        f.write(line + "\n")
    return line

File: jobs/2026-05-27__01-32-13/reminder_sleep_workflow_ts__KTdveAr/artifacts/user/myproject/src/activities.ts
import fs from 'node:fs';

export async function Notify(message: string): Promise<string> {
  const line = `Notified: ${message}`;
  fs.mkdirSync('/workspace', { recursive: true });
  fs.appendFileSync('/workspace/reminder.log', `${line}\n`);
  return line;
}
